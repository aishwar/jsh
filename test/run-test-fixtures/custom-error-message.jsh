
## Custom error message
cd dir-that-does-not-exist #! 404: directory not found

## Log message that you should never reach
echo Done