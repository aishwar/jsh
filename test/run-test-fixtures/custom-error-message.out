[LOG]  Custom error message
[ERROR]  404: directory not found
Program terminated due to error
